### PR TITLE
Added Link in Weekly

### PR DESCRIPTION
**Pull request by YanMagale**
Added link https://umaar.com/dev-tips/, a weekly about features and how to use Chrome Devtools

### CODE REVIEW

File: _posts/2016-09-03-como-se-manter-atualizado-no-frontend.md [hunk 1]
```diff
@@ -110,6 +110,7 @@ Weekly é um conjunto de links, como o nome já diz, semanal. Dos trocentos link
 - [Javascript Weekly](http://javascriptweekly.com/) - weekly totalmente focada em JS.
 - [FiveJS](https://fivejs.codeschool.com/) - patrocinada pela CodeSchool, são 5 vídeos/links sobre JS da semana.
 - [Web Tools Weekly](http://webtoolsweekly.com/) - já pensou numa newsletter falando sobre ferramentas para te ajudar no dia a dia?
+- [Umaar Hansa Dev Tips](https://umaar.com/dev-tips/]) - Uma weekly para acompanhar as novas features e recursos do Devtools do Chrome
```
Comment (willianjusten) [inline _posts/2016-09-03-como-se-manter-atualizado-no-frontend.md]: Fix typo (closing brackets inside link)

Comment (victorferraz) [inline _posts/2016-09-03-como-se-manter-atualizado-no-frontend.md]: I have never trusted in @YanMagale  suggestions.

**Comment by YanMagale** [comment]
Sorry, i dont have see ... fixed @willianjusten

File: _posts/2016-09-03-como-se-manter-atualizado-no-frontend.md (hunk 1 above)
Comment (YanMagale) [inline _posts/2016-09-03-como-se-manter-atualizado-no-frontend.md]: kkkkkkk why not trusted in me @victorferraz ? Believe me :smile:

**Pull request merged**